sync-version/action.yaml

```diff
@@ -36,11 +62,21 @@ runs:
     - name: Commit changes
       if: steps.git-check.outputs.modified == 'true'
       shell: bash
+      with:
+        yaml_path: ${{ steps.yaml-path.outputs.yaml_path }}
+        prev_version: ${{ steps.yaml-path.outputs.prev_version }}
       env:
         SNAP_NAME: ${{ github.event.repository.name }}
       run: |
-        version="$(cat snap/snapcraft.yaml | yq -r '.version')"
+        new_version="$(yq -r '.version' $yaml_path)"
+        if [[ "$prev_version" -eq "$new_version" ]]; then
+            version=$new_version
+        fi
         git config --global user.name 'Snapcrafters Bot'
         git config --global user.email 'merlijn.sebrechts+snapcrafters-bot@gmail.com'
-        git commit -am "chore: bump ${SNAP_NAME} to version $version"
+        if [ $version ]; then
```

Comment:
```suggestion
        if [[ -n "${version:-}" ]]; then
```

Reply:
This is probably outdated, as it'll only work when version has a value set to empty/null. Can you recheck it.

Reply:
Not quite - I made the suggestion for a couple of reasons:

- Prefer `[[` over `[` to match the style of the rest of the file
- `${version:-}` means you can refer to `$version` even if it's previously unset
- `-n` is checking for non-zero length, so if `$version` is non-zero, then we use its content to populate the commit message.